integration/tests/cook/util.py

```diff
@@ -1226,3 +1226,19 @@ def num_hosts_to_consider(cook_url, mesos_url):
     num_hosts = len(hosts_to_consider(cook_url, mesos_url))
     logging.info(f'There are {num_hosts} hosts to consider')
     return num_hosts
+
+
+def should_expect_sandbox_directory(instance=None, job=None):
```

Comment:
Can we have a separate helper function for job that invokes this function which only handles instances?

Reply:
Good idea, done